04-oop-1-class-constructor/assignment-solutions.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
You can also modify the `hasChildren` parameter from the parent file to have that arrow in the menu

Reply:
good catch!